### PR TITLE
feat(manager-components): add <Clipboard /> component

### PR DESCRIPTION
**Pull request by rjamet-ovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-14512
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

Add <Clipboard /> component

## Related

- [x] CMT-13878

### CODE REVIEW

**Review by aboungnaseng-ovhcloud — DISMISSED** [review]

File: packages/manager-components/src/components/clipboard/clipboard.component.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsClipboard, OsdsText } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import './translations';
+
+export interface ClipboardProps {
+  value: string;
+}
```
Comment (chipp972) [inline packages/manager-components/src/components/clipboard/clipboard.component.tsx:9]: Use the Attribute type of OsdsClipboard instead and pass all the props
There an `inline` property for example that we can use

File: packages/manager-components/src/components/clipboard/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "clipboard_copy_success": "Copié !",
```
[the remark this answers is not in the document]
Comment (rjamet-ovh) [reply packages/manager-components/src/components/clipboard/translations/Messages_fr_FR.json:2]: This wording has already been validated

File: packages/manager-components/src/components/clipboard/clipboard.component.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  inline,
+  value,
+}) => {
+  const { t } = useTranslation('clipboard');
+
+  return (
+    <OsdsClipboard
+      value={value}
+      inline={inline}
+      disabled={disabled}
```
Comment (chipp972) [inline packages/manager-components/src/components/clipboard/clipboard.component.tsx:19]:
```suggestion
export const Clipboard: React.FC<OdsClipboardAttribute> = (props) => {
  const { t } = useTranslation('clipboard');

  return (
    <OsdsClipboard
      {...props}
```

If they add or remove props, you won't have to change the code

**Review by chipp972 — DISMISSED** [review]

File: packages/manager-components/src/components/clipboard/translations/index.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import i18next from 'i18next';
+
+import fr_FR from './Messages_fr_FR.json';
+
```
Comment (chipp972) [inline packages/manager-components/src/components/clipboard/translations/index.ts:4]: Don't forget to modify this file once you have the translations to include all the languages

**Review by vovh — DISMISSED** [review]

**Review by darsene — DISMISSED** [review]

**Review by vovh — DISMISSED** [review]

**Pull request merged**